setup.py

```diff
@@ -380,6 +380,7 @@ def __init__(self, dist):
             self.parallel = True
 
     def add(self, ext):
+        print(ext.name)
```

Comment:
Perhaps make this a little bit more verbose?

Reply:
This change should not have landed in git. I accidentally committed debug code.